### PR TITLE
shalinee | test | add amount field code on report , dashboard and pan…

### PR DESCRIPTION
**Pull request by shalinee-walkover**
…ding module

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

**Comment by shalinee-walkover** [comment]
https://app.clickup.com/t/298zrq - 
Developer - @shalinee-walkover 
Description : added amount field code on report, dashboard and panding module

**Note: Will remove commented code once PR gets review.**

File: apps/web-giddh/src/app/home/components/total-overdues/total-overdues-chart.component.html [hunk 1]
```diff
@@ -26,24 +26,48 @@
...
+                            </span>
+                        </span>
                     </div>
                     <div class="clearfix">
                         <small>{{localeData?.pending_invoices}}</small>
-                        <small class="pull-right">{{ ReceivableDurationAmt | giddhCurrency }}</small>
+                        <!-- <small class="pull-right">{{ ReceivableDurationAmt | giddhCurrency }}</small> -->
+                        <small class="d-inline-flex pull-right">
+                            <amount-field [amount]="ReceivableDurationAmt" [currencySymbol]="false"
+                                [currencyCode]="activeCompany.baseCurrency">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/home/components/total-overdues/total-overdues-chart.component.html:43]:
```suggestion
                                [currencyCode]="false">
```

File: apps/web-giddh/src/app/home/components/total-overdues/total-overdues-chart.component.html [hunk 2]
```diff
@@ -26,24 +26,48 @@
...
                         <span class="mr-b1 due-amount">{{localeData?.vendor_due}}</span>
                         <span
-                            class="total-amount pull-right customer-due-text"><span>{{amountSettings.baseCurrencySymbol}}</span>
-                            {{ totalPayable | giddhCurrency }}</span>
+                            class="total-amount pull-right customer-due-text">
+
+                            <!-- <span>{{amountSettings.baseCurrencySymbol}}</span>
+                            {{ totalPayable | giddhCurrency }} -->
+                            <span class="d-inline-flex">
+                                <amount-field [amount]="totalRecievable" [currencySymbol]="amountSettings.baseCurrencySymbol"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/home/components/total-overdues/total-overdues-chart.component.html:57]:
```suggestion
                                <amount-field [amount]="totalPayable" [currencySymbol]="amountSettings.baseCurrencySymbol"
```

File: apps/web-giddh/src/app/home/components/total-overdues/total-overdues-chart.component.html [hunk 3]
```diff
@@ -26,24 +26,48 @@
...
+                            </span>
+                        </span>
                     </div>
                     <div class="clearfix">
                         <small>{{localeData?.pending_invoices}}</small>
-                        <small class="pull-right">{{ PaybaleDurationAmt | giddhCurrency }}</small>
+                        <!-- <small class="pull-right">{{ PaybaleDurationAmt | giddhCurrency }}</small> -->
+                        <small class="d-inline-flex pull-right">
+                            <amount-field [amount]="PaybaleDurationAmt" [currencySymbol]="false"
+                                [currencyCode]="activeCompany.baseCurrency">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/home/components/total-overdues/total-overdues-chart.component.html:68]:
```suggestion
                                [currencyCode]="false">
```

File: apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html [hunk 4]
```diff
@@ -63,9 +63,14 @@ <h5>
...
                             <div>
                                 <span
-                                    class="number-total">{{receiptsSummaryData?.advance?.symbol}}{{receiptsSummaryData?.advance?.amount
+                                    class="number-total d-inline-flex">
+                                    <!-- {{receiptsSummaryData?.advance?.symbol}}{{receiptsSummaryData?.advance?.amount
                                     |
-                                    giddhCurrency}}/-</span>
+                                    giddhCurrency}}/- -->
+
+                                    <amount-field [amount]="receiptsSummaryData?.advance?.amount" [currencySymbol]="receiptsSummaryData?.advance?.symbol" [currencyCode]="false">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html:71]: Send currencyCode as well

File: apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html [hunk 5]
```diff
@@ -74,8 +79,10 @@ <h5>
                             {{localeData?.normal_receipts}}
                             <div>
                                 <span class="d-inline-flex number-total">
-                                    {{receiptsSummaryData?.normal?.symbol}}{{receiptsSummaryData?.normal?.amount |
-                                    giddhCurrency}}/-
+                                    <!-- {{receiptsSummaryData?.normal?.symbol}}{{receiptsSummaryData?.normal?.amount |
+                                    giddhCurrency}}/- -->
+                                    <amount-field [amount]="receiptsSummaryData?.normal?.amount" [currencySymbol]="receiptsSummaryData?.normal?.symbol" [currencyCode]="false">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html:84]: Send currencyCode as well

File: apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html [hunk 6]
```diff
@@ -85,8 +92,12 @@ <h5>
                             {{localeData?.total_receipts}}
                             <div>
                                 <span class="d-inline-flex number-total">
-                                    {{receiptsSummaryData?.total?.symbol}}{{receiptsSummaryData?.total?.amount |
-                                    giddhCurrency}}/-
+                                    <!-- {{receiptsSummaryData?.total?.symbol}}{{receiptsSummaryData?.total?.amount |
+                                    giddhCurrency}}/- -->
+
+                                    <amount-field [amount]="receiptsSummaryData?.total?.amount" [currencySymbol]="receiptsSummaryData?.total?.symbol" [currencyCode]="false">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html:98]: Send currencyCode as well

File: apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html [hunk 7]
```diff
@@ -96,8 +107,10 @@ <h5>
                             {{localeData?.unused_receipts}}
                             <div>
                                 <span class="d-inline-flex number-total">
-                                    {{receiptsSummaryData?.unUsed?.symbol}}{{receiptsSummaryData?.unUsed?.amount |
-                                    giddhCurrency}}/-
+                                    <!-- {{receiptsSummaryData?.unUsed?.symbol}}{{receiptsSummaryData?.unUsed?.amount |
+                                    giddhCurrency}}/- -->
+                                    <amount-field [amount]="receiptsSummaryData?.unUsed?.amount" [currencySymbol]="receiptsSummaryData?.unUsed?.symbol" [currencyCode]="false">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html:112]: Send currencyCode as well

File: apps/web-giddh/src/app/reports/components/columnar-report-table-component/columnar.report.table.component.html [hunk 8]
```diff
@@ -23,11 +23,18 @@
                             <span
                                 *ngIf="isCrDrChecked">{{(columnarReportResponse?.openingBalance[item.accountUniqueName]?.type
                                 ==='DEBIT' && columnarReportResponse?.openingBalance[item.accountUniqueName]?.amount
-                                >0)? '-': ''}} </span> {{
-                            columnarReportResponse?.openingBalance[item.accountUniqueName]?.amount}}
+                                >0)? '-': ''}} </span>
+                                <!-- {{ columnarReportResponse?.openingBalance[item.accountUniqueName]?.amount }} -->
+                                <span class="d-inline-flex">
+                                    <amount-field [amount]="columnarReportResponse?.openingBalance[item.accountUniqueName]?.amount " [currencySymbol]="false" [currencyCode]="false">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/reports/components/columnar-report-table-component/columnar.report.table.component.html:29]:
```suggestion
                                    <amount-field [amount]="columnarReportResponse?.openingBalance[item.accountUniqueName]?.amount" [currencySymbol]="false" [currencyCode]="false">
```

File: apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html (hunk 4 above)
Comment (prateek-giddh) [inline apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html:71]: Do check currencyCode is not passed but currencySymbol is provided which will not make it RTL for Gulf countries.

File: apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html (hunk 5 above)
Comment (prateek-giddh) [inline apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html:84]: Do check currencyCode is not passed but currencySymbol is provided which will not make it RTL for Gulf countries.

File: apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html (hunk 6 above)
Comment (prateek-giddh) [inline apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html:98]: Do check currencyCode is not passed but currencySymbol is provided which will not make it RTL for Gulf countries.

File: apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html (hunk 7 above)
Comment (prateek-giddh) [inline apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html:112]: Do check currencyCode is not passed but currencySymbol is provided which will not make it RTL for Gulf countries.

File: apps/web-giddh/src/app/reports/components/purchase-register-table-component/purchase.register.table.component.html [hunk 9]
```diff
@@ -4,21 +4,73 @@
...
-                <th *ngIf="purchaseRegisterTotal.returns" class="text-right">{{purchaseRegisterTotal.returns |
-                    giddhCurrency}}</th>
-                <th *ngIf="purchaseRegisterTotal.taxTotal" class="text-right">{{purchaseRegisterTotal.taxTotal |
-                    giddhCurrency}}</th>
-                <th *ngIf="purchaseRegisterTotal.discountTotal" class="text-right">{{purchaseRegisterTotal.discountTotal
-                    | giddhCurrency}}</th>
+                <th class="text-right">
+                    <!-- {{purchaseRegisterTotal.purchase | giddhCurrency}} -->
+                    <span class="d-inline-flex">
+                        <amount-field [amount]="purchaseRegisterTotal.purchase " [currencySymbol]="false"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/reports/components/purchase-register-table-component/purchase.register.table.component.html:10]:
```suggestion
                        <amount-field [amount]="purchaseRegisterTotal.purchase" [currencySymbol]="false"
```

File: apps/web-giddh/src/app/reports/components/purchase-register-table-component/purchase.register.table.component.html [hunk 10]
```diff
@@ -4,21 +4,73 @@
...
+                                [currencyCode]="false">
+                        </amount-field>
+                    </span>
+                </th>
+                <th *ngIf="purchaseRegisterTotal.returns" class="text-right">
+                    <!-- {{purchaseRegisterTotal.returns | giddhCurrency}} -->
+                    <span class="d-inline-flex">
+                        <amount-field [amount]="purchaseRegisterTotal.returns " [currencySymbol]="false"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/reports/components/purchase-register-table-component/purchase.register.table.component.html:18]:
```suggestion
                        <amount-field [amount]="purchaseRegisterTotal.returns" [currencySymbol]="false"
```

File: apps/web-giddh/src/app/reports/components/purchase-register-table-component/purchase.register.table.component.html [hunk 11]
```diff
@@ -4,21 +4,73 @@
...
+                                [currencyCode]="false">
+                        </amount-field>
+                    </span>
+                </th>
+                <th *ngIf="purchaseRegisterTotal.taxTotal" class="text-right">
+                    <!-- {{purchaseRegisterTotal.taxTotal | giddhCurrency}} -->
+                    <span class="d-inline-flex">
+                        <amount-field [amount]="purchaseRegisterTotal.taxTotal " [currencySymbol]="false"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/reports/components/purchase-register-table-component/purchase.register.table.component.html:26]:
```suggestion
                        <amount-field [amount]="purchaseRegisterTotal.taxTotal" [currencySymbol]="false"
```

File: apps/web-giddh/src/app/reports/components/purchase-register-table-component/purchase.register.table.component.html [hunk 12]
```diff
@@ -4,21 +4,73 @@
...
+                                [currencyCode]="false">
+                        </amount-field>
+                    </span>
+                </th>
+                <th *ngIf="purchaseRegisterTotal.discountTotal" class="text-right">
+                    <!-- {{purchaseRegisterTotal.discountTotal
+                    | giddhCurrency}} -->
+                    <span class="d-inline-flex">
+                        <amount-field [amount]="purchaseRegisterTotal.discountTotal " [currencySymbol]="false"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/reports/components/purchase-register-table-component/purchase.register.table.component.html:35]:
```suggestion
                        <amount-field [amount]="purchaseRegisterTotal.discountTotal" [currencySymbol]="false"
```

File: apps/web-giddh/src/app/reports/components/purchase-register-table-component/purchase.register.table.component.html [hunk 13]
```diff
@@ -4,21 +4,73 @@
...
+                                [currencyCode]="false">
+                        </amount-field>
+                    </span>
+                </th>
                 <th *ngIf="isTcsTdsApplicable && purchaseRegisterTotal.tcsTotal" class="text-right">
-                    {{purchaseRegisterTotal.tcsTotal | giddhCurrency}}</th>
+                    <!-- {{purchaseRegisterTotal.tcsTotal | giddhCurrency}} -->
+                    <span class="d-inline-flex">
+                        <amount-field [amount]="purchaseRegisterTotal.tcsTotal " [currencySymbol]="false"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/reports/components/purchase-register-table-component/purchase.register.table.component.html:43]:
```suggestion
                        <amount-field [amount]="purchaseRegisterTotal.tcsTotal" [currencySymbol]="false"
```

File: apps/web-giddh/src/app/reports/components/purchase-register-table-component/purchase.register.table.component.html [hunk 14]
```diff
@@ -4,21 +4,73 @@
...
+                </th>
                 <th *ngIf="isTcsTdsApplicable && purchaseRegisterTotal.tdsTotal" class="text-right">
-                    {{purchaseRegisterTotal.tdsTotal | giddhCurrency}}</th>
-                <th *ngIf="purchaseRegisterTotal.netPurchase" class="text-right">{{purchaseRegisterTotal.netPurchase |
-                    giddhCurrency}}</th>
-                <th *ngIf="purchaseRegisterTotal.cumulative" class="text-right">{{purchaseRegisterTotal.cumulative |
-                    giddhCurrency}}</th>
+                    <!-- {{purchaseRegisterTotal.tdsTotal | giddhCurrency}} -->
+                    <span class="d-inline-flex">
+                        <amount-field [amount]="purchaseRegisterTotal.tdsTotal " [currencySymbol]="false"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/reports/components/purchase-register-table-component/purchase.register.table.component.html:51]:
```suggestion
                        <amount-field [amount]="purchaseRegisterTotal.tdsTotal" [currencySymbol]="false"
```

File: apps/web-giddh/src/app/reports/components/purchase-register-table-component/purchase.register.table.component.html [hunk 15]
```diff
@@ -4,21 +4,73 @@
...
+                                [currencyCode]="false">
+                        </amount-field>
+                    </span>
+                </th>
+                <th *ngIf="purchaseRegisterTotal.netPurchase" class="text-right">
+                    <!-- {{purchaseRegisterTotal.netPurchase |
+                    giddhCurrency}} -->
+                    <span class="d-inline-flex">
+                        <amount-field [amount]="purchaseRegisterTotal.netPurchase " [currencySymbol]="false"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/reports/components/purchase-register-table-component/purchase.register.table.component.html:60]:
```suggestion
                        <amount-field [amount]="purchaseRegisterTotal.netPurchase" [currencySymbol]="false"
```

File: apps/web-giddh/src/app/reports/components/purchase-register-table-component/purchase.register.table.component.html [hunk 16]
```diff
@@ -4,21 +4,73 @@
...
+                                [currencyCode]="false">
+                        </amount-field>
+                    </span>
+                </th>
+                <th *ngIf="purchaseRegisterTotal.cumulative" class="text-right">
+                    <!-- {{purchaseRegisterTotal.cumulative |
+                    giddhCurrency}} -->
+                    <span class="d-inline-flex">
+                        <amount-field [amount]="purchaseRegisterTotal.cumulative " [currencySymbol]="false"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/reports/components/purchase-register-table-component/purchase.register.table.component.html:69]:
```suggestion
                        <amount-field [amount]="purchaseRegisterTotal.cumulative" [currencySymbol]="false"
```

File: apps/web-giddh/src/app/reports/components/purchase-register-table-component/purchase.register.table.component.html (hunk 9 above)
Comment (prateek-giddh) [inline apps/web-giddh/src/app/reports/components/purchase-register-table-component/purchase.register.table.component.html:10]:
```suggestion
                        <amount-field [amount]="purchaseRegisterTotal.purchase" [currencySymbol]="false"
```

File: apps/web-giddh/src/app/reports/components/purchase-register-table-component/purchase.register.table.component.html (hunk 10 above)
Comment (prateek-giddh) [inline apps/web-giddh/src/app/reports/components/purchase-register-table-component/purchase.register.table.component.html:18]:
```suggestion
                        <amount-field [amount]="purchaseRegisterTotal.returns" [currencySymbol]="false"
```

File: apps/web-giddh/src/app/reports/components/purchase-register-table-component/purchase.register.table.component.html (hunk 12 above)
Comment (prateek-giddh) [inline apps/web-giddh/src/app/reports/components/purchase-register-table-component/purchase.register.table.component.html:35]:
```suggestion
                        <amount-field [amount]="purchaseRegisterTotal.discountTotal" [currencySymbol]="false"
```

File: apps/web-giddh/src/app/reports/components/purchase-register-table-component/purchase.register.table.component.html (hunk 13 above)
Comment (prateek-giddh) [inline apps/web-giddh/src/app/reports/components/purchase-register-table-component/purchase.register.table.component.html:43]:
```suggestion
                        <amount-field [amount]="purchaseRegisterTotal.tcsTotal" [currencySymbol]="false"
```

File: apps/web-giddh/src/app/reports/components/purchase-register-table-component/purchase.register.table.component.html (hunk 14 above)
Comment (prateek-giddh) [inline apps/web-giddh/src/app/reports/components/purchase-register-table-component/purchase.register.table.component.html:51]:
```suggestion
                        <amount-field [amount]="purchaseRegisterTotal.tdsTotal" [currencySymbol]="false"
```

File: apps/web-giddh/src/app/reports/components/purchase-register-table-component/purchase.register.table.component.html (hunk 15 above)
Comment (prateek-giddh) [inline apps/web-giddh/src/app/reports/components/purchase-register-table-component/purchase.register.table.component.html:60]:
```suggestion
                        <amount-field [amount]="purchaseRegisterTotal.netPurchase" [currencySymbol]="false"
```

File: apps/web-giddh/src/app/reports/components/purchase-register-table-component/purchase.register.table.component.html (hunk 16 above)
Comment (prateek-giddh) [inline apps/web-giddh/src/app/reports/components/purchase-register-table-component/purchase.register.table.component.html:69]:
```suggestion
                        <amount-field [amount]="purchaseRegisterTotal.cumulative" [currencySymbol]="false"
```

File: apps/web-giddh/src/app/reports/components/reverse-charge-report-component/reverse-charge-report.component.html [hunk 17]
```diff
@@ -61,9 +61,14 @@
...
                             {{localeData?.total_taxable_value}}
                             <div>
-                                <span class="d-inline-flex number-total">
+                                <!-- <span class="d-inline-flex number-total">
                                     {{activeCompany?.baseCurrencySymbol}}
                                     {{reverseChargeReportResults?.totalTaxableValue | giddhCurrency }}/-
+                                </span> -->
+                                <span class="d-inline-flex number-total">
+                                    <amount-field [amount]="reverseChargeReportResults?.totalTaxableValue" [currencySymbol]="activeCompany?.baseCurrencySymbol"
+                                            [currencyCode]="activeCompany.baseCurrency">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/reports/components/reverse-charge-report-component/reverse-charge-report.component.html:70]:
```suggestion
                                            [currencyCode]="activeCompany?.baseCurrency">
```

File: apps/web-giddh/src/app/reports/components/reverse-charge-report-component/reverse-charge-report.component.html [hunk 18]
```diff
@@ -72,9 +77,14 @@ <h5>
                         <h5>
                             {{localeData?.total_tax_amount}}
                             <div>
-                                <span class="d-inline-flex number-total">
+                                <!-- <span class="d-inline-flex number-total">
                                     {{activeCompany?.baseCurrencySymbol}} {{reverseChargeReportResults?.totalTaxAmount |
                                     giddhCurrency}}/-
+                                </span> -->
+                                <span class="d-inline-flex number-total">
+                                    <amount-field [amount]="reverseChargeReportResults?.totalTaxAmount" [currencySymbol]="false"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/reports/components/reverse-charge-report-component/reverse-charge-report.component.html:85]:
```suggestion
                                    <amount-field [amount]="reverseChargeReportResults?.totalTaxAmount" [currencySymbol]="activeCompany?.baseCurrencySymbol"
```

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/reports/components/reverse-charge-report-component/reverse-charge-report.component.html (hunk 18 above)
Comment (prateek-giddh) [inline apps/web-giddh/src/app/reports/components/reverse-charge-report-component/reverse-charge-report.component.html:85]: Do check currencySymbol is not provided but currencyCode is provided. Check this for Gulf country company

File: apps/web-giddh/src/app/home/components/total-overdues/total-overdues-chart.component.html [hunk 19]
```diff
@@ -26,24 +26,48 @@
...
+                                </amount-field>
+                            </span>
+                        </span>
                     </div>
                     <div class="clearfix">
                         <small>{{localeData?.pending_invoices}}</small>
-                        <small class="pull-right">{{ PaybaleDurationAmt | giddhCurrency }}</small>
+                        <!-- <small class="pull-right">{{ PaybaleDurationAmt | giddhCurrency }}</small> -->
+                        <small class="d-inline-flex pull-right">
+                            <amount-field [amount]="PaybaleDurationAmt" [currencySymbol]="false"
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/home/components/total-overdues/total-overdues-chart.component.html:67]: Do check currencySymbol is not provided but currencyCode is provided. Check this for Gulf country company

File: apps/web-giddh/src/app/home/components/total-overdues/total-overdues-chart.component.html [hunk 20]
```diff
@@ -26,24 +26,48 @@
...
+                                </amount-field>
+                            </span>
+                        </span>
                     </div>
                     <div class="clearfix">
                         <small>{{localeData?.pending_invoices}}</small>
-                        <small class="pull-right">{{ ReceivableDurationAmt | giddhCurrency }}</small>
+                        <!-- <small class="pull-right">{{ ReceivableDurationAmt | giddhCurrency }}</small> -->
+                        <small class="d-inline-flex pull-right">
+                            <amount-field [amount]="ReceivableDurationAmt" [currencySymbol]="false"
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/home/components/total-overdues/total-overdues-chart.component.html:42]: Do check currencySymbol is not provided but currencyCode is provided. Check this for Gulf country company

**Review by prateek-giddh — CHANGES_REQUESTED** [review]
@shalinee-walkover Please remove all the commented code in all the files of this PR

File: apps/web-giddh/src/app/home/components/total-overdues/total-overdues-chart.component.html (hunk 1 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/home/components/total-overdues/total-overdues-chart.component.html:43]: done

File: apps/web-giddh/src/app/home/components/total-overdues/total-overdues-chart.component.html (hunk 2 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/home/components/total-overdues/total-overdues-chart.component.html:57]: done

File: apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html (hunk 4 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html:71]: done

File: apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html (hunk 5 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html:84]: activeCompany.baseCurrency

File: apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html (hunk 6 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html:98]: Done

File: apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html (hunk 7 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html:112]: done

File: apps/web-giddh/src/app/reports/components/columnar-report-table-component/columnar.report.table.component.html (hunk 8 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/reports/components/columnar-report-table-component/columnar.report.table.component.html:29]: Done

File: apps/web-giddh/src/app/reports/components/purchase-register-table-component/purchase.register.table.component.html (hunk 9 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/reports/components/purchase-register-table-component/purchase.register.table.component.html:10]: Done

File: apps/web-giddh/src/app/reports/components/purchase-register-table-component/purchase.register.table.component.html (hunk 10 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/reports/components/purchase-register-table-component/purchase.register.table.component.html:18]: Done

File: apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html (hunk 4 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html:71]: added

File: apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html (hunk 5 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html:84]: done

File: apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html (hunk 6 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html:98]: done

File: apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html (hunk 7 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html:112]: Done

File: apps/web-giddh/src/app/reports/components/reverse-charge-report-component/reverse-charge-report.component.html (hunk 18 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/reports/components/reverse-charge-report-component/reverse-charge-report.component.html:85]: added

File: apps/web-giddh/src/app/home/components/total-overdues/total-overdues-chart.component.html (hunk 3 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/home/components/total-overdues/total-overdues-chart.component.html:68]: done

File: apps/web-giddh/src/app/home/components/cr-dr-list/cr-dr-list.component.html [hunk 21]
```diff
@@ -39,13 +39,24 @@
...
+                                            [currencyCode]="activeCompany.baseCurrency">
+                                        </amount-field>
+                                    </span>
+                                </td>
                                 <td [attr.data-title]="commonLocaleData?.app_due_as_on" class="text-right"
                                     *ngIf="!account.latestBillAmount">-</td>
                                 <td [attr.data-title]="commonLocaleData?.app_due_as_on" class="text-right">
-                                    {{activeCompany.baseCurrencySymbol}} {{ account.closingBalance.amount |
-                                    giddhCurrency }}</td>
+                                    <!-- {{activeCompany.baseCurrencySymbol}} {{ account.closingBalance.amount |
```
[the remark this answers is not in the document]
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/home/components/cr-dr-list/cr-dr-list.component.html:53]: REmove all commented code

**Review by prateek-giddh — APPROVED** [review]

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**